google/cloud/spanner_v1/session.py

```diff
@@ -114,7 +115,11 @@ def create(self):
         kw = {}
         if self._labels:
             kw = {"session": {"labels": self._labels}}
-        session_pb = api.create_session(self._database.name, metadata=metadata, **kw)
+
+        with trace_call("CloudSpanner.CreateSession", self, self._labels):
```

Comment:
I wonder do we have a uniform format for name? 

- In Go, we have a format like `cloud.google.com/go/spanner.ReadWriteTransaction` (https://github.com/googleapis/google-cloud-go/blob/e143982c0485e1ff2726f7e2b607c26997dd0d90/spanner/client.go#L423). 
- In Java, we have a format "CloudSpannerOperation.Commit" (https://github.com/googleapis/java-spanner/blob/462839b625e58e235581b8ba10b398e1d222eaaf/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java#L59-L67). 

I guess it is okay to use this format because every language has a different format.

Reply:
Go is a bit special. Can you match it with `Java`? Thanks for the update.